ullyses/coadd.py

```diff
@@ -13,7 +13,7 @@
 
 
 STIS_NON_CCD_DETECTORS = ['FUV-MAMA', 'NUV-MAMA']
-BAD_SEGMENTS = ['NUVC']
+BAD_SEGMENTS = {2635: 'NUVC', 2950: 'NUVC'}
```

Comment:
I think this should go in the wrapper code - ULLYSES and HSLA might have different opinions on whether data is 'bad'
In wrapper.py, add the BAD_SEGMENTS definition to the top of the Ullyses_SegmentList (just before the write method).  Then in coadd.py, refer to self.BAD_SEGMENTS.  HSLA can have its own BAD_SEGMENTS definition in their wrapper, so it doesn't get locked in to the coadd coade that both projects share

Reply:
Done 👍 I added a check in coadd to see if the `bad_segments` attribute exists, so it's not necessary to add to subclasses. I ran the wrapper script with some G230L/2950 data and confirmed it excluded NUVC data.